tasks/main.yml

```
@@ -1,39 +1,45 @@
 ---
 
-- name: Get os specific facts
-  ansible.builtin.include_vars: "{{ item }}"
+- name: get os specific facts
+  ansible.builtin.include_vars: '{{ item }}'
   with_first_found:
     - '{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.yml'
     - '{{ ansible_distribution }}.yml'
     - '{{ ansible_os_family }}.yml'  
 
-- name: get Wallpapers to remove
+- name: get wallpapers to remove
   ansible.builtin.shell:
-    cmd: "ls -p /usr/share/backgrounds | grep -v"
+    cmd: ls -p /usr/share/backgrounds | grep -v
   register: potos_wallpaper_backgrounds
   changed_when: false
 
-- name: remove Wallpapers
+- name: remove wallpapers
   ansible.builtin.file:
-    path: "/usr/share/backgrounds/{{ item }}"
+    path: '/usr/share/backgrounds/{{ item }}'
     state: absent
-  loop: "{{ potos_wallpaper_backgrounds | difference( potos_wallpaper_images ) }}"
+  loop: '{{ potos_wallpaper_backgrounds | difference( potos_wallpaper_images ) }}'
```

To remote remove warning from pipeline:

```suggestion
  loop: '{{ potos_wallpaper_backgrounds | difference(potos_wallpaper_images) }}'
```